presentation/common/src/presentation-common/content/Fields.ts

```diff
@@ -46,6 +46,8 @@ export interface NestedContentFieldJSON extends BaseFieldJSON {
   contentClassInfo: ClassInfoJSON;
   pathToPrimaryClass: RelationshipPathJSON;
   /** @alpha */
+  relationshipMeaning?: string;

```

Comment:
The type should be `RelationshipMeaning`, right?

Reply:
Changed

Reply:
still a `string` in `NestedContentFieldJSON`